zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -361,11 +412,13 @@ where
 
                 size_limit: MAX_BLOCK_BYTES,
 
-                cur_time: 0,
+                cur_time: cur_time.timestamp(),
```

Comment:
I changed the type of `cur_time` in my other suggestion.

```suggestion
                cur_time,
```